Mathlib/NumberTheory/LSeries/Dirichlet.lean

```diff
@@ -0,0 +1,415 @@
+/-
+Copyright (c) 2024 Michael Stoll. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Michael Stoll
+-/
+import Mathlib.NumberTheory.DirichletCharacter.Bounds
+import Mathlib.NumberTheory.LSeries.Convolution
+import Mathlib.NumberTheory.LSeries.Deriv
+import Mathlib.NumberTheory.SumPrimeReciprocals
+import Mathlib.NumberTheory.VonMangoldt
+import Mathlib.NumberTheory.ZetaFunction
+
+/-!
+# L-series of Dirichlet characters and arithmetic functions
+
+We collect some results on L-series of specific (arithmetic) functions, for example,
+the Möbius function `μ` or the von Mangoldt function `Λ`. In particular, we show that
```

Comment:
As a rule, I would prefer spelling names as they are written in the person's language (at least when it uses latin script). I think the "oe" spelling was used historically because "ö" was not readily avaiable on non-German typewriters/keyboards, but nowadays we have basically all variations of latin letters available, and so there is really no good reason anymore to not write "Möbius" (or "Gödel", "Gröbner", "Teichmüller", "Gauß", "Poincaré" etc.).

I'd be willing to at least submit a PR standardizing "Möbius" throughout Mathlib.

Reply:
(In declaration names, it is a different matter, of course.)